### ISSUE TITLE
Investigate reports of COD payment method not showing up for users of Woo Blocks 3.0.0

### ISSUE THREAD

**Issue by nerrad** [woocommerce/woocommerce-gutenberg-products-block#2910]
We're getting reports in the [release post](https://woocommerce.wordpress.com/2020/07/22/woocommerce-blocks-3-0-release-notes/) of folks not seeing the COD payment method available for checkout block. I've been unable to reproduce yet but thought I'd create this issue to track these reports and give a place for folks experiencing this to add information about how the payment method is configured on their end to see if we can get some reproducible steps.

**Comment by nerrad** [comment]
I'm so far not able to reproduce this on my test site with the WC Blocks 3.0.0 downloaded from wp.org installed and active. Here's what I've tested:

- Activated COD payment method.
- Tried various restrictions for the COD payment method options (and no restrictions).
- Tested while logged in and logged out, in both cases COD (and across variations of COD settings) the payment method appeared and I was able to checkout.

Until we get more details from folks who don't see the payment method as an option, there's not much that can be done here.
